notificationworkerlambda/cdk/lib/senderworker.ts

```diff
@@ -110,13 +111,13 @@ class SenderWorker extends cdkcore.Construct  {
       memorySize: 10240,
       description: `sends notifications for ${id}`,
       role: executionRole,
-      timeout: cdk.Duration.seconds(180),
+      timeout: props.isBatchingSqsMessages ? cdk.Duration.seconds(180) : cdk.Duration.seconds(90),
       reservedConcurrentExecutions: props.reservedConcurrency
     })
 
     const senderSqsEventSourceMapping = new lambda.EventSourceMapping(this, "SenderSqsEventSourceMapping", {
-      batchSize: 20,
-      maxBatchingWindow: cdk.Duration.seconds(1),
+      batchSize: props.isBatchingSqsMessages ? 20 : 1,
+      maxBatchingWindow: props.isBatchingSqsMessages ? cdk.Duration.seconds(1) : cdk.Duration.seconds(0),
```

Comment:
I _think_ this is needed? I wondered if the default was 0 if it wasn't specified, but if we only gett 1 sqs message per event, maybe the maxBatchingWindow is irrelevant anyway

Reply:
I agree!